app/serializers/product_shallow_serializer.rb

```diff
@@ -21,6 +25,6 @@ def url
   cached
 
   def cache_key
-    object
+    ['v2', object]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.